test/forms/common/AddressSpec.scala

```diff
@@ -203,16 +167,21 @@ object AddressSpec {
 
   import play.api.libs.json._
 
+  val seventyOneChars = "01234567890123456789012345678901234567890123456789012345678901234567890"
+  val thirtySixChars = "0123456789012345678901234567890123456"
+  val tenChars = "01234567890"
```

Comment:
Nit: There is a `TestHelper` class that contains `createRandomString` method. You can simply provide the length you want, no need to type it yourself ;)